<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>247</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Добавить точку</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>Замкнуть фигуру</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Выбрать цвет</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Закрасить</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Границ</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Фона</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>Заливки</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Задержка</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="table"/>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Удалить точку</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Затравочный пиксель</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="canvas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
